### ISSUE TITLE
Installing package from local repository that has diacritic in the path fails

### ISSUE THREAD

**Issue by AsciiWolf** [flatpak/flatpak#1817]
## Linux distribution and version
Fedora 28
## Flatpak version
0.11.8.3
## Description of the problem
Installing package from local Flatpak repository fails if there is a non-ASCII character somewhere in the directory path.

For example:
```
Installing for user: com.jagex.RuneScape/x86_64/master from runerepo
error: Failed to install com.jagex.RuneScape/x86_64/master: /home/asciiwolf/Sta%C5%BEen%C3%A9/com.jagex.RuneScape/runerepo: opendir(/home/asciiwolf/Sta%C5%BEen%C3%A9/com.jagex.RuneScape/runerepo): No such file or directory
```
## Steps to reproduce
1. Create a directory with some diacritic characters in its name.
1. Create a local Flatpak repository inside the created directory. (You can download some Git repo from Flathub and use `flatpak-builder` to build it.)
2. Add the repository using a `flatpak remote-add` command.
3. Try to install a package from this repository.

**Comment by alexlarsson** [comment]
Yeah, it looks like it incorrectly roundtrips via a url (or at least url escapes) somewhere...
